Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItem.swift

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Simple form item that represent a single selectable element.
+@_spi(AdyenInternal)
+public class SelectableFormItem: FormItem, Hidable {
+
+    public var subitems: [FormItem] = []
+
+    /// The title of the item.
+    public var title: String
+
+    /// The icon to display
+    public var icon: Icon?
+
+    /// Determines whether the item is currently the selected one.
+    @AdyenObservable(false) public var isSelected: Bool
+
+    /// The handler to invoke when the item is selected.
+    public var selectionHandler: (() -> Void)?
+
+    /// The `accessibilityIdentifier` to be used on the `SelectableFormItem`
+    public var identifier: String?
+
+    /// The `accessibilityLabel` to be used on the ``SelectableFormItem``
+    public let accessibilityLabel: String
+
+    /// The selectable item style.
+    public let style: SelectableFormItemStyle
+
+    public var isHidden: AdyenObservable<Bool> = AdyenObservable(false)
+
+    /// Initializes the list item.
```

Comment:
not list item :)

Reply:
8079ceeba101a6e70258f18631cbfa9367c7b89e